salt/reboot/init.sls

```diff
@@ -0,0 +1,42 @@
+##################################################
+# Configuration for the reboot manager
+##################################################
+
+# `max_holders` contains the maximum number of lock holders for the cluster.
+{% set max_holders = salt['pillar.get']('etcd:masters', '1')|int %}
+{% if max_holders > 1 %}
+  {% set max_holders = max_holders - 1 %}
+{% endif %}
+
+# Cleanup any previous cluster information on /opensuse.org
+opensuseorg_cleanup:
+  pkg.installed:
+    - name: etcdctl
+  cmd.run:
+    - name: etcdctl
+            rm -r /opensuse.org
+    # ignore failures for this
+    - check_cmd:
+      - /bin/true
```

Comment:
We fail to connect to etcd, or something related. The idea is that salt doesn't consider it a fail, always consider it a success, even if `etcdctl` returned a non-zero status.